docroot/themes/humsci/humsci_basic/src/scss/components/_main-nav.scss

```diff
@@ -1,105 +1,354 @@
+
   &__toggle {
-    border-width: 0 !important;
+    @include unbuttonize;
+    cursor: pointer;
     position: absolute;
     right: 0;
     top: hb-calculate-rems(-120px);
 
-    // Hamburger Icon
-    &::after {
-      background-image: svg('<svg viewBox="0 0 24 24"><path d="M0 4.13575C0 3.23201 0.732625 2.49939 1.63636 2.49939H22.3636C23.2674 2.49939 24 3.23201 24 4.13575C24 5.03949 23.2674 5.77212 22.3636 5.77212H1.63636C0.732625 5.77212 0 5.03949 0 4.13575Z" fill="#2e2d29"/><path d="M0 19.4085C0 18.5047 0.732625 17.7721 1.63636 17.7721H22.3636C23.2674 17.7721 24 18.5047 24 19.4085C24 20.3122 23.2674 21.0448 22.3636 21.0448H1.63636C0.732625 21.0448 0 20.3122 0 19.4085Z" fill="#2e2d29"/><path d="M0 11.7721C0 10.8684 0.732625 10.1357 1.63636 10.1357H22.3636C23.2674 10.1357 24 10.8684 24 11.7721C24 12.6758 23.2674 13.4085 22.3636 13.4085H1.63636C0.732625 13.4085 0 12.6758 0 11.7721Z" fill="#2e2d29"/></svg>') !important;
+    font-weight: hb-theme-font-weight(bold);
+
+    display: flex;
+    align-items: center;
+    width: hb-calculate-rems(70px);
+    flex-direction: initial;
+    padding: hb-calculate-rems(10px) 0;
+
+    .no-js & {
+      display: none;
+    }
+
+    @include grid-media-min('lg') {
+      .js & {
+        display: none;
+      }
+    }
+
+    .hb-main-nav__toggle-icon {
+      @include hb-menu-hamburger-icon;
+      margin-left: hb-calculate-rems(10px);
     }
 
-    // Close Icon
-    &[aria-expanded=true]::after {
-      background-image: svg('<svg viewBox="0 0 24 24"><path d="M10.56,12,.32,22.21a1,1,0,0,0,1.41,1.41L12,13.39,22.21,23.62a1,1,0,0,0,1.41-1.41L13.39,12,23.62,1.74a1,1,0,0,0,0-1.42,1,1,0,0,0-1.41,0L12,10.56,1.74.32A1,1,0,0,0,.32,1.74Z" fill="#2e2d29" transform="translate(-0.03 -0.03)"/></svg>') !important;
+    &[aria-expanded="true"] .hb-main-nav__toggle-icon {
+      @include hb-menu-close-icon;
     }
   }
 
   &__menu {
+    // scss-lint:disable ImportantRule
     box-shadow: none !important;
     transform: translate(0) !important;
     width: auto !important;
+    // scss-lint:enable ImportantRule
 
     list-style-type: none;
     margin: 0;
     padding: 0;
 
-    line-height: 2;
-
     @include grid-media-min('sm') {
       max-width: none;
     }
 
+    &[aria-hidden="true"] {
+      display: none;
+    }
+
     &-lv1 {
-      position: relative !important;
+      font-weight: hb-theme-font-weight(bold);
+      font-size: hb-calculate-rems(22px);
+      border-bottom: $hb-menu-light-border;
 
-      &[aria-hidden="true"] {
-        display: none !important;
+      &[aria-hidden="false"] {
+        margin-bottom: hb-calculate-rems(28px);
       }
 
-      @include grid-media('md') {
-        display: flex;
-        max-width: none !important;
+      @include grid-media-min('md') {
+        font-size: hb-calculate-rems(30px);
       }
 
-      @include grid-media('lg') {
-        .su-main-nav__menu-container {
-          position: absolute;
+      @include grid-media-min('lg') {
+        .js & {
```

Comment:
can you add a little comment here about why the .js class is used and what it's showing

Reply:
Since that class pops up a few times in the file I added a note to the top of the file.